3-Authorization-II/2-call-api-b2c/API/authConfig.js

```diff
@@ -0,0 +1,33 @@
+const passportConfig = {
+    credentials: {
+        tenantName: 'fabrikamb2c.onmicrosoft.com',
+        clientID: 'e29ac359-6a90-4f9e-b31c-8f64e1ac20cb',
+    },
+    policies: {
+        policyName: 'B2C_1_susi',
+    },
+    metadata: {
+        b2cDomain: 'fabrikamb2c.b2clogin.com',
+        authority: 'login.microsoftonline.com',
+        discovery: '.well-known/openid-configuration',
+        version: 'v2.0',
+    },
+    settings: {
+        isB2C: true,
+        validateIssuer: false,
+        passReqToCallback: false,
+        loggingLevel: 'info',
```

Comment:
should have another setting here: 

`loggingNoPII`: false

and we should empahsize in README and app.js that they can set this to true to for logging and debugging.